nss-cohort-36/c36-class-website
# Lr class website

# Description
Please run the json server and then run npm start on your terminal. You should see the about section in you browser.

## Type of change
Please delete options that are not relevant.
# Testing Instructions
Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout lr-class-website`
4. please run your json server on 5002 and run npm start on your terminal as well
# Checklist:
